bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/screens/DigitalInvoiceScreen.kt

```diff
@@ -60,14 +58,20 @@ class DigitalInvoiceScreen {
                 .className("android.widget.ImageButton")
                 .descriptionContains("Close")
         )
-        if(cancelButton.isClickable() && cancelButton.exists()){
+        if(cancelButton.exists() && cancelButton.isClickable()){
             cancelButton.click()
         }
     }
 
-    fun assertOtherChargesDisplayed() {
-      onView(withText(net.gini.android.bank.sdk.R.string.gbs_digital_invoice_addon_other_charges)).check(
-            matches(isDisplayed()))
+    fun assertOtherChargesDisplayed() : Boolean {
+        var isOtherChargesDisplayed = false
+        onView(withText(net.gini.android.bank.sdk.R.string.gbs_digital_invoice_addon_other_charges))
+            .check { view, noViewFoundException ->
+                if (noViewFoundException == null || view.isShown()) {
```

Comment:
[Clarification] I think here `isOtherChargesDisplayed` should be `true` if there is no exception AND the view is shown, right? 🤔 

Or do we want to return `isOtherChargesDisplayed` as `true` even if the view is not shown?

Reply:
Hmm, I will try replacing `OR` with `AND` to see if the condition is okay.
If not, I will make the changes as we've discussed.

Reply:
1. When `OR` is replaced with `AND`
```
 fun assertOtherChargesDisplayed() : Boolean {
        var isOtherChargesDisplayed = false
        onView(withText(net.gini.android.bank.sdk.R.string.gbs_digital_invoice_addon_other_charges))
            .check { view, noViewException ->
                if (view.isShown() && noViewException == null) {
                    isOtherChargesDisplayed = true
                }
            }
        return isOtherChargesDisplayed
    }
```

Reply:
@danicretu which among the two listed approach I should be using?

Reply:
As discussed we're going with point `2`.